news-sentiment/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Analyze the Sentiment of News Headlines
+description: A Google Sheet with Apps Script that Analyzes the sentiment of news headlines retrieved for a user specified topic from a free News API.
+labels: Apps Script, Sheets, Cloud Natural Language
+material_icon: Feed
+create_time: 2021-03-22
+update_time: 2021-03-22
+---
+
+This solution consists of a Google Sheet that uses Apps Script to fetch the current news headlines from the free news API `newsapi.org` using a user provded `topic`.
+
+Once the news headlines are fetched and loaded onto the sheet, it runs a Natural Language sentiment detection for each headline. 
```

Comment:
Don't capitalize natural language.

Reply:
Ack. I only capitalize when mentioning the product name "Google Cloud Natural Language API". I removed the non product reference.